code/modules/supermatter/supermatter.dm

```diff
@@ -99,18 +99,18 @@
 	. = ..()
 
 /obj/machinery/power/supermatter/proc/explode()
-	var/turf/turff = get_turf(src)
+	if(istype(universe,/datum/universal_state/supermatter_cascade))
```

Comment:
So, if we're already in a cascade, you send us into a cascade, and the supermatter only actually explodes if that happens?